packages/synapse-react-client/src/synapse-queries/github/useGithub.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { SynapseClientError, useSynapseContext } from '../../utils'
+
+const GITHUB_API_URL = 'https://api.github.com/repos'
+
+export function useLatestTag(
```

Comment:
This react-query wrapped fetch to get the latest tag will be reused in the new ToS workflow (where we will need to provide this to the backend).  We will also reuse the MarkdownGithub component (to get and display the new ToS content).